kythe/go/extractors/config/preprocessor/Dockerfile

```diff
@@ -0,0 +1,3 @@
+FROM scratch
```

Comment:
this is definitely up for changing later, for now just want to get something up there that I can test with

In particular I'm not so keen on this approach because it'll probably rely on mounting volumes to pass input/output file back and forth.  I'm still a bit fuzzy on how cloud build handles that.

Reply:
actually, it turns out this just works as-is in cloud build.

I was wrong, custom volume mapping logic will not be necessary here.